recipes/rsync/all/conanfile.py

```diff
@@ -6,7 +6,7 @@
 from conan.tools.apple import is_apple_os
 import os
 
-required_conan_version = ">=1.60.0"
+required_conan_version = ">=2.4"
```

Comment:
Why is this needed? Please consider reverting this as the current recipe still works fine with Conan 1.x.

Reply:
Thanks a lot @perseoGI 🚀